README.yaml

```diff
@@ -48,41 +48,41 @@ references:
   - name: "Terraform Module Requirements"
```

Comment:
Add links to 
- https://www.terraform.io/docs/cloud/workspaces/agent.html
- https://www.hashicorp.com/blog/announcing-hashicorp-terraform-cloud-business'
- https://www.hashicorp.com/resources/announcing-terraform-cloud-business-tier-with-armon-dadgar